latex/delegation.tex

```diff
@@ -333,19 +430,70 @@
 
 \end{figure}
 
+
+The rules for updating the part of the ledger state defining the current stake
+pools are given in \cref{fig:pool-rules}. Note
+that in all three rules, the current epoch is part of the context. It is not
+necessary to include $cepoch$ in the context for \cref{eq:pool-reg}. However,
+we decided to include it for consistency and composability of stake pool rules.
+
+For each rule, again, we first check that a given certificate $c$ is of
+the correct type using the
+corresponding $\RegPool$ and $\RetirePool$ predicates.
+
+The first rule, \cref{eq:pool-reg}, can be used to register a new stake pool, register a
+stake pool under a different author ($\fun{author} c$), or stop the retirement
+process for that key (the hash key and associated planned retirement epoch
+ will be removed from the $retiring$ parameter).
+
+The second rule, \cref{eq:pool-ret}, starts the pool retirement process. Given an
+epoch $e$, the application of this rule requires that this epoch is in
+the future (after $cepoch$), as well as that it is
+less than $\emax$ epochs after the current one, and also that
+the certificate author's key is in the list of registered pools.
+This rule simply adds the pair of the key of the
+certificate author and the planned retirement epoch $e$ to the $retiring$
+parameter.
+
+The third rule, \cref{eq:pool-reap}, is used when the planned retirement epoch $e$ for
+a given stake pool in the $\var{retiring}$ set is reached. In this case, all the
+pairs associated with hash keys of pools which are scheduled to retire in this
+epoch are removed
+from both $\var{stpools}$ and $\var{retiring}$.
+
 \subsection{Witnesses}
 \label{sec:delegation-witnesses}
```

Comment:
This would be a good place to briefly mention that the delegation witnesses piggyback off the replay attack prevention of the UTxO ledger by signing the transaction body.